scipy/stats/_qmc.py

```diff
@@ -862,25 +856,30 @@ def __init__(
         self.base = n_primes(d)
         self.scramble = scramble
 
-    def random(self, n: IntNumber = 1) -> np.ndarray:
+    def random(self, n: IntNumber = 1, workers: IntNumber = 1) -> np.ndarray:
```

Comment:
Make it keyword only.
```suggestion
    def random(self, n: IntNumber = 1, *, workers: IntNumber = 1) -> np.ndarray:
```